src/Icons/Icons.tsx

```diff
@@ -124,6 +128,20 @@ const AutoPolygonIcon: React.FC<Props> = ({ size = 'large', fill = DEFAULT_ICON_
 );
 export { AutoPolygonIcon, autoPolygonUrl };
 
+const LeaseholdAutoPolygonIcon: React.FC<Props> = ({ size = 'large', fill = DEFAULT_ICON_COLOR, ...props }) => (
+  <IconWrapper size={size}>
+    <LeaseholdAutoPolygon fill={fill} {...props} />
+  </IconWrapper>
+);
+export { LeaseholdAutoPolygonIcon, autoPolygonUrl as leaseholdAutoPolygonUrl };
```

Comment:
I think you can just rename `AutoPolygonIcon` rather than create a duplicate with a different name.

Reply:
You'll need to add it to the stories file too so it shows up in Stroybook, which I'm happy to help with if you need it

Reply:
I'd prefer not to change the name, as it would introduce a breaking change to the code base. This slow API migration means we can remove the old `AutoPolygonIcon` component once we're happy.

Reply:
So that icon isn't used anywhere else (only in maps), but that's fine if we want to approach it like that, can you add a note to remove the duplicate though as we tend to forget about these things.

Reply:
The app code can be changed to use the new name when updating the wombat version in Maps, so it shouldn’t be too risky in any case